verification/curator-service/src/index.ts

```diff
@@ -1,12 +1,14 @@
-import express from 'express';
+import bodyParser from 'body-parser';
```

Comment:
Wonder if there's a linter option/plugin for managing import order.

Reply:
Ah, that was done by vs code, not sure if there's a linter plugin but I can look around for it...